client/controllers/incidentReports/incidentReport.coffee

```diff
@@ -15,11 +15,13 @@ Template.incidentReport.helpers
   caseCounts: ->
     @deaths or @cases
 
-  deathsLabel: ->
-    pluralize 'Death', @deaths, false
-
-  casesLabel: ->
-    pluralize 'Case', @cases, false
+  countLabel: ->
+    if @type == 'activeCount'
+      pluralize 'Active Case', @cases, false
+    else if @cases
+      pluralize 'Case', @cases, false
+    else if @deaths
+      pluralize 'Death', @deaths, false
```

Comment:
I think we should wrap the arguments in parens. I think the style guide recommends doing so if they can all fit on one line.